packages/flutter/lib/src/painting/borders.dart

```diff
@@ -294,6 +307,62 @@ class BorderSide {
     );
   }
 
+  /// Get the amount of the stroke width that lies inside of the [BorderSide].
+  ///
+  /// This will return 0 for a [strokeAlign] of 1, and the width of the stroke for
+  /// a [strokeAlign] of -1.
+  ///
+  /// Simplified example:
```

Comment:
What does this example show?  It doesn't seem related to `strokeInset`, and I'm not sure when I would copy and use that code as a developer.

Reply:
I change it to:
```
/// Example of results:
/// StrokeAlign.inside (-1) => width
/// StrokeAlign.center (0) => width / 2
/// StrokeAlign.outside (1) => 0
```
What do you think? The main idea was to show how progressing the values change the output in a less-secret way.